### DISCUSSION TITLE
Can markdown be imported at runtime?

### DISCUSSION THREAD

**Discussion by asv7c2** [pngwn/MDsveX#226] [Q&A]
I have a blog with more than 900 posts in markdown. To decrease time building (i use sveltekit and adapter-static adapter) i want to load those markdown at runtime. Is that possible? Or better go with adapter-node?

**Comment by pngwn** [comment]
For the time being the best option is to use SSR with `adapter-node` for this kind of thing. I hope to have a better solution for this in the future.
